src/custom_clojure_ttt/winning_move_handler.clj

```diff
@@ -0,0 +1,7 @@
+(ns custom-clojure-ttt.winning_move_handler
+  (:require [clojure-tic-tac-toe.default_winning_moves :as default_winning_moves]))
+
+(defn get-winning-moves
+  [side-length]
+  default_winning_moves/winning-moves)
```

Comment:
I understand that you only have the winning moves for a 3x3 board at the moment but since you're building out this project so that in the future, you can add functionality for a custom board size, would it make sense to pass in the side length to a function on your other project's namespace?

Reply:
I'm not sure I fully understand. The data for different board sizes will be contained in the custom project, so doesn't it make sense to pass in the side-length here?